lib/vehicle_factory.rb

```diff
@@ -0,0 +1,19 @@
+class VehicleFactory
+  
+  def initialize
+  end
+
+  def create_vehicles(location)
+    new_car_list = []
```

Comment:
Same thing here, I would want this array held in state, so I could refer back to it instead of having to run this method everytime I want to access these vehicles.